ts/screens/preferences/ServiceDetailsScreen.tsx

```diff
@@ -47,6 +47,7 @@ type Props = ReturnType<typeof mapStateToProps> & ReduxProps & OwnProps;
 
 interface State {
   uiEnabledChannels: EnabledChannels;
+  updateStatus: pot.Pot<true, string>;
```

Comment:
from the logic below it looks like `this.props.profile` follows exactly the lifecycle of `this.props.profile` ? can you check whether the problem is solved by just disabling the update while `pot.isLoading(this.props.profile)` ? in theory `this.props.profile` should tell you when the profile is updating - perhaps is the logic in the saga that handles the `NewEnabledChannels` action that is not updating `profile` correctly